### PR TITLE
Code style: black

### PR DESCRIPTION
**Pull request by asottile**
Other than `README.md`, `tox.ini`, and `.pre-commit-config.yaml` this commit is entirely automated -- I had to convince flake8 that 88 characters (black's default) is ok.  When I tried to set black to lower than 88 characters it refused to reflow some strings :/

### CODE REVIEW

File: src/bandersnatch/main.py [hunk 1]
```diff
@@ -66,60 +69,82 @@ def mirror(config):
...
     subparsers = parser.add_subparsers()
 
     # `mirror` command
     m = subparsers.add_parser(
-        'mirror',
-        help='Performs a one-time synchronization with '
-             'the PyPI master server.')
-    m.set_defaults(op='mirror')
+        "mirror",
+        help="Performs a one-time synchronization with " "the PyPI master server.",
```
Comment (cooperlees) [inline src/bandersnatch/main.py]: This is one `black` thing I dislike. But get why hard for the AST to detect.
Please join the str

File: src/bandersnatch/utils.py [hunk 2]
```diff
@@ -88,17 +87,19 @@ def update_safe(filename: str, **kw: Any) -> Generator[IO, None, None]:
 
     """
     with tempfile.NamedTemporaryFile(
-            dir=os.path.dirname(filename), delete=False,
-            prefix=os.path.basename(filename) + '.', **kw) as tf:
+        dir=os.path.dirname(filename),
+        delete=False,
+        prefix=os.path.basename(filename) + ".",
```
Comment (cooperlees) [inline src/bandersnatch/utils.py]: NIT: could f"" - No big deal.

File: tox.ini [hunk 3]
```diff
@@ -27,6 +27,9 @@ deps=pre-commit
 commands=
     pre-commit run --all-files --show-diff-on-failure
 
+[flake8]
+max_line_length = 88
```
Comment (cooperlees) [inline tox.ini]: 👍

File: src/bandersnatch/utils.py (hunk 2 above)
Comment (asottile) [reply src/bandersnatch/utils.py]: I'll follow up with https://github.com/asottile/pyupgrade for some free f-string goodness :D

File: .pre-commit-config.yaml [hunk 4]
```diff
@@ -8,6 +8,11 @@ repos:
     -   id: check-added-large-files
     -   id: flake8
         exclude: ^docs/conf.py$
+-   repo: https://github.com/ambv/black
+    rev: 18.6b2
```
Comment (cooperlees) [inline .pre-commit-config.yaml]: Is there any way to just say latest version?

**Comment by cooperlees** [comment]
Also, if you want to chat - I sit in #bandersnatch on freenode

File: .pre-commit-config.yaml (hunk 4 above)
Comment (asottile) [reply .pre-commit-config.yaml]: [intentionally not](https://pre-commit.com/#using-the-latest-version-for-a-repository) -- speed and repeatability trade-offs were made.

Fortunately `pre-commit autoupdate` is a quick and easy way to get to the latest :)

File: src/bandersnatch/main.py (hunk 1 above)
Comment (asottile) [reply src/bandersnatch/main.py]: fixed, and in the 4 other places this happened too

File: src/bandersnatch/tests/test_utils.py [hunk 5]
```diff
@@ -7,44 +7,45 @@
 
 
 def test_hash():
```
Comment (Isaac9710009) [inline src/bandersnatch/tests/test_utils.py]: src/bandersnatch/configuration.py

**Pull request merged**